src/Common/DateLUTImpl.h

```diff
@@ -712,11 +712,11 @@ class DateLUTImpl
     /// (round down to monday and divide DayNum by 7; we made an assumption,
     ///  that in domain of the function there was no weeks with any other number of days than 7)
     template <typename DateOrTime>
-    Int32 toRelativeWeekNum(DateOrTime v) const
+    Int32 toRelativeWeekNum(DateOrTime v, UInt8 day_of_week_mode = 1) const
```

Comment:
Let's avoid defaults. They create 10% more convenience in the PR but increase horrors for everyone else trying to change something in the code later on.